src/routes/Missions.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getMissons, reserveToggle } from '../redux/missions/missionSlice';
+
+const Missions = () => {
+  const { missions, isLoading, error } = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMissons());
+  }, [dispatch]);
```

Comment:
- [ ] The same issue discussed earlier is happening on the Missions page too. Please do the same thing here to resolve it. ✍️